From 76b8f0df65c979bf802d5cb19de1456e321942ef Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 26 May 2020 11:24:44 +0800
Subject: [PATCH 3/3] gpu: mail-400: use PTR_ERR_OR_ZERO instead of PTR_RET

Because of mainline commit fad7c9020948 ("err.h: remove deprecated
PTR_RET for good"), use PTR_ERR_OR_ZERO instead of PTR_RET.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 linux/mali_memory_dma_buf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux/mali_memory_dma_buf.c b/linux/mali_memory_dma_buf.c
index 9cdd48a..a46ced3 100644
--- a/linux/mali_memory_dma_buf.c
+++ b/linux/mali_memory_dma_buf.c
@@ -281,7 +281,7 @@ int mali_dma_buf_get_size(struct mali_session_data *session, _mali_uk_dma_buf_ge
 	buf = dma_buf_get(fd);
 	if (IS_ERR_OR_NULL(buf)) {
 		MALI_DEBUG_PRINT_ERROR(("Failed to get dma-buf from fd: %d\n", fd));
-		return PTR_RET(buf);
+		return PTR_ERR_OR_ZERO(buf);
 	}
 
 	if (0 != put_user(buf->size, &user_arg->size)) {
-- 
2.17.1

